taxi/views.py

```diff
@@ -1,3 +1,18 @@
+from django.http import HttpResponse, HttpRequest
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Car, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
```

Comment:
No reason to create this variables, you can specify values in context dict directly